packages/grafana-ui/src/components/CustomScrollbar/CustomScrollbar.tsx

```diff
@@ -140,3 +143,52 @@ export class CustomScrollbar extends Component<Props> {
 }
 
 export default CustomScrollbar;
+
+const getStyles = (page?: boolean) => {
+  return {
+    customScrollbar: css`
+      display: flex;
+      flex-grow: 1;
+      &:hover {
+        opacity: 0.8;
+        transition: opacity 0.3s ease-in-out;
+      }
+    `,
+    trackVertical: css`
+      border-radius: 3px;
+      width: 8px !important;
+      right: ${page ? 0 : 2}px;
+      bottom: 2px;
+    `,
+    trackHorizontal: css`
+      border-radius: 3px;
+      height: 8px !important;
+      right: 2px;
+      bottom: 2px;
+      left: 2px;
+    `,
+    thumbVertical: css`
+      @include gradient-vertical($scrollbarBackground, $scrollbarBackground2);
+      border-radius: 6px;
+      opacity: 0;
+      &:hover {
+        opacity: 0.8;
+        transition: opacity 0.3s ease-in-out;
+      }
+    `,
+    thumbHorizontal: css`
+      @include gradient-horizontal($scrollbarBackground, $scrollbarBackground2);
+      border-radius: 6px;
+      opacity: 0;
+      &:hover {
+        opacity: 0.8;
+        transition: opacity 0.3s ease-in-out;
+      }
+    `,
+    view: css`
+      display: flex;
+      flex-grow: 1;
+      flex-direction: column;
+    `,
+  };
+};
```

Comment:
nit: I think you should add a return type to this function. Then you should use that return type in the functions above instead of using `styles: any`. Types help you avoid unforced errors at transpile time instead of runtime. They're your best TypeScript friend :)
```suggestion
interface XYZ {
 customScrollbar: xyz,
 ...
 ...
}
const getStyles = (page?: boolean): XYZ => {
  return {
    customScrollbar: css`
      display: flex;
      flex-grow: 1;
      &:hover {
        opacity: 0.8;
        transition: opacity 0.3s ease-in-out;
      }
    `,
    trackVertical: css`
      border-radius: 3px;
      width: 8px !important;
      right: ${page ? 0 : 2}px;
      bottom: 2px;
    `,
    trackHorizontal: css`
      border-radius: 3px;
      height: 8px !important;
      right: 2px;
      bottom: 2px;
      left: 2px;
    `,
    thumbVertical: css`
      @include gradient-vertical($scrollbarBackground, $scrollbarBackground2);
      border-radius: 6px;
      opacity: 0;
      &:hover {
        opacity: 0.8;
        transition: opacity 0.3s ease-in-out;
      }
    `,
    thumbHorizontal: css`
      @include gradient-horizontal($scrollbarBackground, $scrollbarBackground2);
      border-radius: 6px;
      opacity: 0;
      &:hover {
        opacity: 0.8;
        transition: opacity 0.3s ease-in-out;
      }
    `,
    view: css`
      display: flex;
      flex-grow: 1;
      flex-direction: column;
    `,
  };
};
```